libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java

```diff
@@ -558,6 +559,10 @@ public void setTitle(CharSequence text) {
         mTitle.setText(text);
     }
 
+    @NonNull @Override public String getEditorName() {
```

Comment:
NIT: move these to separate lines?

Reply:
Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/10794/commits/5341b48d43033f78d44d7ea5baf62249f6f78898